.travis.yml

```diff
@@ -11,9 +11,11 @@ before_install:
     - 'if [[ $GROUP != js* ]]; then COVERAGE=""; fi'
 install:
     - pip install -f travis-wheels/wheelhouse -r requirements.txt -e file://$PWD#egg=ipython[test] coveralls
+    - pip install codecov
```

Comment:
> makes no sense, but OK.

Yeah, No clue why either. I'll put that on travis weird infrastructure, 
it's failling only only on 3.5, but not for me locally.